tests/foreman/api/test_subscription.py

```diff
@@ -126,7 +126,7 @@ def test_negative_upload(self):
         :expectedresults: The manifest is not uploaded to the second
             organization.
 
-        :CaseImportance: Critical
+        :CaseImportance: High
```

Comment:
Since you have the same value on module level, you don't need to change this line - you could just remove it.
Of course leaving it is fine, too.

Reply:
removing is a better option